bundles/org.openhab.binding.lgthinq/src/main/java/org/openhab/binding/lgthinq/lgservices/model/devices/fridge/FridgeCapabilityFactoryV1.java

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.lgthinq.lgservices.model.devices.fridge;
+
+import static org.openhab.binding.lgthinq.lgservices.LGServicesConstants.CAP_RE_FRESH_AIR_FILTER_MAP;
+import static org.openhab.binding.lgthinq.lgservices.LGServicesConstants.CAP_RE_ON_OFF;
+import static org.openhab.binding.lgthinq.lgservices.LGServicesConstants.CAP_RE_SMART_SAVING_MODE;
+import static org.openhab.binding.lgthinq.lgservices.LGServicesConstants.CAP_RE_WATER_FILTER_USED_POSTFIX;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.lgthinq.lgservices.model.CommandDefinition;
+import org.openhab.binding.lgthinq.lgservices.model.FeatureDefinition;
+import org.openhab.binding.lgthinq.lgservices.model.LGAPIVerion;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * The {@link FridgeCapabilityFactoryV1}
+ *
+ * @author Nemer Daud - Initial contribution
+ */
+@NonNullByDefault
+public class FridgeCapabilityFactoryV1 extends AbstractFridgeCapabilityFactory {
+
+    @Override
+    protected FeatureDefinition newFeatureDefinition(String featureName, JsonNode featuresNode,
+            @Nullable String targetChannelId, @Nullable String refChannelId) {
+        // TODO - Implement feature definition
```

Comment:
Possible to fix?

Reply:
Feature definition is a device property/sensor that has a special feature (selectable values, default value definition, special action changing the state, etc). This comment was a default one justo to remember to come back in that points and review this features at the beginning of the biding developing. I will revise to see if there are others to remove.